components/BottomNavigationBar.tsx

```diff
@@ -0,0 +1,50 @@
+"use client"
+
+import { Icon } from '@iconify/react';
+import { usePathname } from 'next/navigation';
+import AvantiLogo from '../assets/avanti_logo.png';
+import Image from 'next/image';
+import NavLink from './NavLink';
+
+const BottomNavigationBar = () => {
+  const pathname = usePathname();
+
+  const iconStyle = {
+    fontSize: '32px',
+    color: 'black',
+  };
+
+  const activeIconStyle = {
+    fontSize: '32px',
+    color: '#008181',
+  };
+
+  const isActive = (path: string) => {
+    return pathname === path;
+  };
+
+  return (
+    <div className="fixed bottom-0 left-0 right-0 text-xs md:text-lg pl-16 pr-16 bg-white pb-4 pt-2 flex justify-between border-t-2 items-center shadow-2xl shadow-black">
+      <NavLink href="/library" active={isActive('/library')}>
+        <Icon
+          icon="ic:round-library-books"
+          style={isActive('/library') ? activeIconStyle : iconStyle}
+        />
+        Library
```

Comment:
how do I test this out?
I want to test that if the route is /library or /reports, I'll see the active icon style css at these elements, but when I try to go to those URLs, it shows me an error. 

The reports one works but even in that page I dont see the nav bar. The nav bar should be visible in all pages/routes right?

Reply:
It is because the page for library is not being created yet. I'll add the screenshot how it will be going to look like in the description of the PR.

For the reports page, currently that page is WIP so it is not using this component for now. We need to add this component in order to make it visible in all our pages.

Reply:
Got it. Understood how it will look like. Also, will you be adding navbar to the reports page in this PR?
I think we should add it in this PR itself. Whatever the future structure of the reports page be, the navbar should be visible across all pages and look the same.